AndroidAnnotations/androidannotations-core/androidannotations-testutils/src/main/java/org/androidannotations/testutils/ProcessorTestHelper.java

```diff
@@ -293,6 +293,10 @@ public CompileResult compileFiles(Object... elements) {
 				addCollection(files, (Type) element);
 			} else if (element instanceof String) {
 				files.add(new File((String) element));
+			} else if (element instanceof String[]) {
```

Comment:
Why is this needed?

Reply:
@WonderCsabo , this comes with the function I explained above, "defPath", which basically returns an array of strings (multiple paths), so on this one, that is simply parsed to get each one of the paths and add them to the files.

Reply:
I understand, thanks.